src/test/pythonnode_test.cpp

```diff
@@ -435,6 +435,72 @@ TEST_F(PythonFlowTest, PythonNodeInitMembers) {
     }
 }
 
+TEST_F(PythonFlowTest, PythonNodePassInitArguments) {
+    ConstructorEnabledModelManager manager;
+    std::string testPbtxt = R"(
+    input_stream: "IMAGE:in"
+    input_stream: "MYTYPE:in2"
+    output_stream: "IMAGE:out"
+    output_stream: "MYTYPE:out2"
```

Comment:
Let's stay in line with calculator expectations to not create confusion in the future. Every tag should start with OVMS_PY_TENSOR.